test/libraries/Sign.sol

```diff
@@ -16,6 +16,8 @@ library Sign {
 
     // Copied from GPv2Signing.sol
     uint256 internal constant PRE_SIGNED = uint256(keccak256("GPv2Signing.Scheme.PreSign"));
+    // As prescribed by EIP-1271
+    bytes4 internal constant EIP1271_MAGIC_VALUE = bytes4(keccak256("isValidSignature(bytes32,bytes)"));
```

Comment:
Is this necessary here as we could use `EIP1271Verifier.isValidSignature.selector` ?